src/cookbook/work-with-non-mintable-assets.md

```diff
@@ -11,4 +11,48 @@ head:
 
 # How to Work with Non-Mintable Assets
 
-TODO
\ No newline at end of file
+```rust
+fn register_non_mintable_asset(
+    iroha: &Client,
+) {
+    let magical_keys = AssetDefinitionId::from_str("magical_key#wonderland").unwrap();
+    // register keys as an asset definition
+    let register_keys_as_a_concept = Register::asset_definition(
+        AssetDefinition::new(
+            magical_keys.clone(),
+            AssetValueType::Numeric(NumericSpec::integer()),
+        ).mintable_once()
```

Comment:
This snippet registers an asset with `mintable_once`, which is different from a non-mintable asset.

Reply:
I took the example from the integration tests: https://github.com/hyperledger/iroha/blob/v2.0.0-pre-rc.21.1/client/tests/integration/non_mintable.rs#L12